src/preregister.js

```diff
@@ -3,22 +3,21 @@ import NavBar from "./nav-bar.js";
 import MailchimpInput from "./mailchimp-input.js";
 import CovidPopover from "./covid-popover.js";
 
-var HeroImage = require('./hero.svg');
+var heroImage = require('./hero.svg');
+var heroImageAlt = "A decorative hero graphic."
 
-var desktopNotice = "COVID-19 Notice: "
-var mobileNotice = "COVID-19 Notice"
 var noticeContent = "We're planning BrickHack 7 to be the best and safest event possible. We will publish relevant information upon receiving updates from RIT and NY State."
 
 class PreRegister extends React.Component {
     render() {
         return (
         <div id="app">
-            <div className="full-height">
-                <NavBar
-                    title={desktopNotice}
-                    content={noticeContent}
-                />
-                <div className="section flex-container">
+            <NavBar 
+                content={noticeContent}
+            />
+
+            <div className="section">
+                <div className="content flex-container">
                     <div className="left-half">
                         <div id="hackathon-name">
                             BRICKHACK 7
```

Comment:
Same issue here, where this text isn't in a `h1` or something, the browser can handle it, but it's not guaranteed what the browser will do with it. 
For example, it isn't clear if the "Brickhack 7" text is being treated as a block-level element or not. If it was in an `h1`, it would be clear that it's block level. If `p`, it's not block level.